capellambse/svg/drawing.py

```diff
@@ -666,6 +665,8 @@ def _draw_box(
             features=features_,
             children=bool(children_),
         )
+        if context_:
+            grp.add(container.Group(class_=" ".join(context_)))
```

Comment:
I don't think putting the raw, undecorated IDs into the `class` attribute is a good idea. Is there a better place for this? HTML has its freely usable `data-X` attributes - does SVG maybe have something similar? If not, I'd prefer to see some prefix added to these IDs, to make it more explicit what they're meant to represent and why they're there.

Reply:
Doesn't work. We tried some attributes, not only data-X. Maybe we didn't try it correctly? SVG-write didn't support this.

Reply:
SVGs have data-* attributes, which we tried, as well as custom namespaces. But it wasn't supported by svgwrite.

Reply:
Okay, so no data attributes - in that case, please add a short prefix like `context-` to the IDs, to make it more apparent what they are about.